src/public/modules/forms/helpers/csv-response-classes/TableResponse.class.ts

```diff
@@ -0,0 +1,25 @@
+import { AnswerArray, DisplayedResponse } from '../../../../../types/response'
+
+import { Response } from './Response.class'
+
+export interface TwoDimResponse extends Omit<DisplayedResponse, 'answerArray'> {
+  answerArray: Extract<AnswerArray, string[][]>
+}
+
+export class TableResponse extends Response {
+  constructor(responseData: TwoDimResponse) {
+    super(responseData)
+  }
+
+  getAnswer(colIndex: number): string {
+    // Leave cell empty if number of rows is fewer than the index
+    if (colIndex >= this._data.answerArray.length) {
+      return ''
+    }
+    return (this._data as TwoDimResponse).answerArray[colIndex].join(';')
```

Comment:
`this._data` should be typed as a class member. I think that should work if you get its type to inherit from `DisplayedResponseWithoutAnswer`